lta/rest_server.py

```diff
@@ -0,0 +1,92 @@
+"""
+Long Term Archive REST API server.
+
+Run with `python -m lta.rest_server`.
+"""
+
+import asyncio
+import logging
+
+from tornado.ioloop import IOLoop
+
+from rest_tools.client import json_decode
```

Comment:
```suggestion
```

Reply:
I'm going to need this shortly, so the import stays.